### ISSUE TITLE
Use -exec-timeout in our internal launch scripts

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#1540]
The current launching scripts appear to still use the old "-timeout" option causing the warning message:

"-timeout option will soon be deprecated, Use -idle-timeout or -exec-timeout"

To be shown from each engine instance launched (in parallel you get one warning per processor)

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Cyrus Harrison
Original creation: 10/14/2010 08:06 pm
Original update: 10/20/2010 01:24 am
Ticket number: 429
